example/movies/ai_assistants.py

```diff
@@ -28,31 +44,38 @@ class IMDbScraper(AIAssistant):
         "You're a function to find the IMDb URL of a given movie, "
         "and scrape this URL to get the movie rating and other information.\n"
         "Use the search function to find the IMDb URL. "
-        "Make search queries like: \n"
-        "- IMDb page of The Matrix\n"
-        "- IMDb page of The Godfather\n"
-        "- IMDb page of The Shawshank Redemption\n"
-        "Then check results, scape the IMDb URL, process the page, and produce a JSON output."
+        "Make search queries like:\n"
+        "- IMDb page of <queried movie here>\n"
+        "Then check results, scrape the IMDb URL, process the page, and produce an output like this: \n"
+        "- IMDb URL: ...\n"
+        "- IMDb Rating: ...\n"
+        "- IMDb Page: <Markdown content of the IMDb page>"
     )
     name = "IMDb Scraper"
     model = "gpt-4o-mini"
-    structured_output = IMDbMovie
+    tool_max_concurrency = 4
```

Comment:
With RateLimitedBraveSearch, we can run tools concurrently and get faster responses. Thanks to this, we can have responses overall from 5 to 30 seconds.

Reply:
Nice!